### ISSUE TITLE
"Telegram API - 400 Bad Request: chat not found" - specifically when adding another Telegram user in the recipients list

### ISSUE THREAD

**Issue by ranshn** [mluggy/teslabox#17]
I have Telegram notifications working after following the tutorial, but I'm unable to add another Telegram user to be notified. For the other Telegram user, I fetched its ID like I did for the first user, put it with a comma in the "Telegram recipients" list, but am getting the 400 error in the logs. I tried removing the first (working) initial Telegram chat ID and only leaving the extra one, but still the same result.

Am I missing a step for adding multiple Telegram recipients?

**Comment by mluggy** [comment]
Hi. Make sure the other user has started a conversation with the bot you created as well.

**Comment by ranshn** [comment]
Yeah I tried sending normal messages or commands from the other user to the bot, but it doesn't reply. Actually, it also doesn't reply to the "working" user (not sure it should though).

**Comment by mluggy** [comment]
it shouldn't reply that's fine, are you sure you have the right userid for both?
at a matter of fact I never tried multiple recipients but if the latter doesn't work even on its own it must be that he hasn't approved the bot.

**Comment by miaszh** [comment]
> I have Telegram notifications working after following the tutorial, but I'm unable to add another Telegram user to be notified. For the other Telegram user, I fetched its ID like I did for the first user, put it with a comma in the "Telegram recipients" list, but am getting the 400 error in the logs. I tried removing the first (working) initial Telegram chat ID and only leaving the extra one, but still the same result.
> 
> 
> 
> Am I missing a step for adding multiple Telegram recipients? 

Sorry for the silly question, did you actually put:
"111111, 222222"?

**Comment by ranshn** [comment]
Started working for some reason... I sent a message again to the bot from the other user, added only the other user to the recipient list and it worked. Added the initial user and it's all working. thanks
